pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -1023,16 +1023,12 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 		return nil, errors.Wrap(err, "failed to add rule opening traffic pod to pod")
 	}
 	for _, cidr := range podSubnetCidrs {
-		err = ensureOpenStackSgRule(client, masterSgId, cidr, -1, -1)
-		if err != nil {
-			return nil, errors.Wrapf(err, "failed to add rule opening traffic to masters on %s", cidr)
-		}
 		err = ensureOpenStackSgRule(client, workerSgId, cidr, -1, -1)
```

Comment:
Why would the traffic to the workers be still open?

Reply:
The console pods require traffic to be open on the worker, and the etcd pods required traffic to be open on the master.